src/components/capture/create/index.tsx

```diff
@@ -228,128 +191,81 @@ function NewCaptureModal() {
     };
 
     return (
-        <Dialog
-            open
-            onClose={handlers.close}
-            scroll="paper"
-            fullScreen={fullScreen}
-            fullWidth={!fullScreen}
-            maxWidth="md"
-            sx={{
-                '.MuiDialog-container': {
-                    alignItems: 'flex-start',
-                },
-            }}
-            aria-labelledby="new-capture-dialog-title"
-        >
-            <DialogTitle id="new-capture-dialog-title">
-                <FormattedMessage id="captureCreation.heading" />
-                <IconButton
-                    aria-label="close"
-                    onClick={handlers.close}
+        <PageContainer>
+            <Toolbar>
+                <Typography variant="h6" noWrap>
+                    <FormattedMessage id="captureCreation.heading" />
+                </Typography>
+
+                <Stack
+                    direction="row"
+                    alignItems="center"
                     sx={{
-                        color: (buttonTheme) => buttonTheme.palette.grey[500],
-                        position: 'absolute',
-                        right: 0,
+                        ml: 'auto',
                     }}
                 >
-                    <CloseIcon />
-                </IconButton>
-            </DialogTitle>
-
-            {formSubmitError && (
-                <NewCaptureError title={formSubmitError.message} errors={[]} />
-            )}
-
-            <DialogContent dividers>
-                {activeStep === Steps.DETAILS_AND_SPEC ? (
```

Comment:
Most of the code is identical. The main change here is removing all the `Dialog` stuff and the `Steps` stuff. But that altered the indentation and that made the changes look bigger than they are.